frontend/src/pages/Login.jsx

```
@@ -0,0 +1,74 @@
+                            name="username"
+                            placeholder="Enter your username"
+                            className="input input-bordered w-full"
+                            value={formData.username}
+                            onChange={handleChange}
+                            required
+                        />
+                    </div>
+                    <div className="form-control mt-2">
+                        <label className="label">
+                            <span className="label-text">Password</span>
+                        </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)